package-lock.json

```diff
@@ -1,8 +1,2090 @@
 {
   "name": "alpha-site",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
```

Comment:
Is something going on here? I didn't expect a 2000+ line change to the package-lock.json for adding a Dockerfile.

Reply:
I believe when I merged in main and fixed the conflict it may have restructured the file. I am not sure why the diff is so large myself.

Reply:
I ended up just copying the lock from main. Pushing up now.